### PR TITLE
zh-cn: update `ReadableStreamDefaultReader.releaseLock()` method

### PR DESCRIPTION
**Pull request by fwqaaq**

### CODE REVIEW

File: files/zh-cn/web/api/readablestreamdefaultreader/releaselock/index.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ slug: Web/API/ReadableStreamDefaultReader/releaseLock
 
 如果释放锁时关联流出错，reader 随后会以同样的方式发生错误；此外，reader 将关闭。
 
-reader 的锁在仍有待处理的读取请求时无法释放，即，如果 reader 的 {{domxref("ReadableStreamDefaultReader.read()")}} 方法返回的 promise 尚未完成，这将导致抛出 `TypeError`。
+reader 的锁在仍有待处理的读取请求时无法释放，那么 reader 的 {{domxref("ReadableStreamDefaultReader.read()")}} 方法返回的 promise 将立刻使用 `TypeError` 拒绝。未读的分块将会保留在内部队列中，并且稍后获取一个新的 reader 读取。
```
Comment (fwqaaq) [inline files/zh-cn/web/api/readablestreamdefaultreader/releaselock/index.md:12]:
```suggestion
reader 的锁在仍有待处理的读取请求时无法释放，那么 reader 的 {{domxref("ReadableStreamDefaultReader.read()")}} 方法返回的 promise 将立刻使用 `TypeError` 拒绝。未读的分块将会保留在内部队列中，并且稍后可以通过获取一个新的 reader 读取。
```

File: files/zh-cn/web/api/readablestreamdefaultreader/releaselock/index.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ slug: Web/API/ReadableStreamDefaultReader/releaseLock
 
 如果释放锁时关联流出错，reader 随后会以同样的方式发生错误；此外，reader 将关闭。
 
-reader 的锁在仍有待处理的读取请求时无法释放，即，如果 reader 的 {{domxref("ReadableStreamDefaultReader.read()")}} 方法返回的 promise 尚未完成，这将导致抛出 `TypeError`。
+reader 的锁在仍有待处理的读取请求时无法释放，那么 reader 的 {{domxref("ReadableStreamDefaultReader.read()")}} 方法返回的 promise 将立刻使用 `TypeError` 拒绝。未读的分块将会保留在内部队列中，并且稍后可以通过获取一个新的 reader 读取。
```
Comment (yin1999) [inline files/zh-cn/web/api/readablestreamdefaultreader/releaselock/index.md:12]:
```suggestion
如果在 reader 仍有待处理的读取请求时释放了锁，那么 reader 的 {{domxref("ReadableStreamDefaultReader.read()")}} 方法返回的 promise 将立刻使用 `TypeError` 拒绝。未读的分块将会保留在内部队列中，并且稍后可以通过获取一个新的 reader 读取。
```

**Review by yin1999 — COMMENTED** [review]
nit:

**Review by yin1999 — APPROVED** [review]
:+1:

**Pull request merged**